packages/experimental/Switch/src/Switch.styling.ts

```diff
@@ -91,6 +92,7 @@ export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, Sw
           width: tokens.thumbSize,
           borderRadius: tokens.thumbRadius,
           margin: tokens.thumbMargin,
+          ...(Platform.OS === 'android' && { elevation: tokens.elevation }),
```

Comment:
Can this take a shadow alias token from fluent? If not we should see if we can change things to be able to do that, for example by using the Shadow component.